missions/templates/missions/defaultassignment_list.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base-missions.html" %}
+{% load bootstrap4 %}
+{% load fontawesome_5 %}
+
+{% block content %}
+<h1>Default Assignments</h1>
+{% if object_list %}
+<table class="table table-hover">
+    <thead>
+        <tr>
+            <th>#</th>
+            <th>Position</th>
+            <th>Quantity</th>
+            <th></th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for default_assignment in object_list %}
+        <tr>
+            <td>{{ default_assignment.id }}</td>
+            <td>{{ default_assignment.position }}</td>
+            <td>{{ default_assignment.quantity }}</td>
+            <td>
+                <a href="{% url 'default_assignment_update' ship.id default_assignment.id %}">{% fa5_icon 'edit' 'fas' %}</a>
+                <a class="text-danger" href="{% url 'default_assignment_delete' ship.id default_assignment.id %}">{% fa5_icon 'trash' 'fas' %}</a>
```

Comment:
Ich würde hier (und bei allen anderen Buttons) noch hilfe toggles einbinden. Siehe z.B. data-toggle="tooltip" title="edit" in registration_process.html

Reply:
Macht sinn